src/pages/LandingPage.js

```diff
@@ -0,0 +1,27 @@
+import 'App.css';
+import 'style.css';
+import { useState, useEffect } from 'react';
+import { http } from 'http';
+import GameContainer from 'components/GameContainer';
+
+function LandingPage() {
+  const [games, setGames] = useState([]);
+
+  useEffect(async () => {
+    const response = await http('https://api.pandascore.co/videogames');

```

Comment:
Hmm I wonder if it would be usefull to have a PandaScore.js inside the `http` folder where you can put the Urls and name them in a bit more descriptive way. e.g

```javascrip
// PandaScore.js
export const ALL_GAMES = 'https://api.pandascore.co/videogames'
```
Then this could be
```suggestion
    const response = await http(ALL_GAMES);
```